### PR TITLE
miscellaneous: Add translations

### PR DESCRIPTION
**Pull request by FabulousCupcake**
This PR adds translations to miscellaneous flavor texts. Credits to `Chris Foryst#0131`.

### CODE REVIEW

File: src/data/miscellaneous.csv [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
-"中",""
-"中～長",""
-"短",""
-"短～中",""
-"長",""
+"短","Short"
+"短～中","Short–Medium"
+"中","Medium"
+"中～長","Medium–Long"
+"長","Long"
```
Comment (FabulousCupcake) [inline src/data/miscellaneous.csv:8]: Need to check if this has strict length limit and will display properly

File: src/data/miscellaneous.csv [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 "162 – School grade/level"
-"中等部",""
-"高等部",""
+"中等部","Middle School"
+"高等部","High School"
```
Comment (FabulousCupcake) [inline src/data/miscellaneous.csv:12]: These too

File: src/data/miscellaneous.csv (hunk 1 above)
Comment (stardustichi) [reply src/data/miscellaneous.csv:8]: where are these in the game? i cant seem to find them

File: src/data/miscellaneous.csv (hunk 2 above)
Comment (FabulousCupcake) [reply src/data/miscellaneous.csv:12]: These are okay

File: src/data/miscellaneous.csv (hunk 1 above)
Comment (FabulousCupcake) [reply src/data/miscellaneous.csv:8]: Yeah it doesn't seem to be used; removing them

File: src/data/miscellaneous.csv [hunk 3]
```diff
@@ -1,19 +1,12 @@
...
 
 "177 – Lottery rewards"
-"温泉旅行券",""
-"特上にんじんハンバーグ",""
-"にんじん山盛り",""
-"にんじん1本",""
-"ティッシュ",""
+"温泉旅行券","Onsen Ticket"
+"特上にんじんハンバーグ","Special Hamburg Carrot"
```
Comment (FabulousCupcake) [inline src/data/miscellaneous.csv:9]: `Special Carrot Steak` according to @robflop

Not sure what is the best 🤔

**Comment by FabulousCupcake** [comment]
Merging for now; we can fix carrot steak laters

**Pull request merged**